test/e2e/elasticsearch_test.go

```diff
@@ -0,0 +1,135 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"time"
+
+	"github.com/ViaQ/log-exploration-api/pkg/configuration"
+	"github.com/ViaQ/log-exploration-api/pkg/constants"
+	"github.com/ViaQ/log-exploration-api/pkg/elastic"
+	"github.com/ViaQ/log-exploration-api/pkg/logs"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var esRepository logs.LogsProvider
+
+func TestMain(t *testing.T) {
+	log, _ := initCustomZapLogger("info")
+
+	appConf := configuration.ParseArgs()
+	esRepository, _ = elastic.NewElasticRepository(log.Named("elasticsearch"), appConf.Elasticsearch)
+}
+
+func TestGetAllLogs(t *testing.T) {
+	repository := esRepository
+	logList, err := repository.GetAllLogs()
+	if err != nil {
```

Comment:
It would be nice to print an error before calling `t.Fail()` so we'd know why a test failed. Consider using `t.Error(...)` (https://golang.org/pkg/testing/#T.Error)

Reply:
Okay @igor-karpukhin, thanks for the review. I'll update it ASAP.